extensions/github/src/components/WorkflowActions.tsx

```diff
@@ -0,0 +1,134 @@
+  repository: string,
+  branch: string,
+  inputs: any,
+  octokit: Octokit
+) {
+  await showToast({ style: Toast.Style.Animated, title: "Sending run request" });
+
+  const [owner, repo] = repository.split("/");
+
+  try {
+    await octokit.rest.actions.createWorkflowDispatch({
+      owner: owner,
+      repo: repo,
+      workflow_id: workflow.id,
+      ref: branch,
+      inputs: inputs,
+    });
+
+    await showToast({
+      style: Toast.Style.Success,
+      title: "Sent run request",
+    });
+  } catch (error) {
+    await showToast({
+      style: Toast.Style.Failure,
+      title: "Failed sending re-run request",
+      message: getErrorMessage(error),
+    });
+  }
+}
+
+type WorkflowActionsProps = {
+  workflow: Workflow;
+  workflows: Workflow[];
+  repository: string;
+  branches: any[];
+  defaultBranch: string;
+  favorites: Workflow[];
+  onUpdateFavorites: (favorites: Workflow[]) => void;
+  mutateList: MutatePromise<WorkflowsResponse | undefined>;
+};
+
+function getURLPathExceptLastPathComponent(url: string): string {
+  return url.split("/").slice(0, -1).join("/");
+}
+
+function replaceStringWithStringInPath(path: string, stringToReplace: string, replacement: string): string {
+  return path.replace(stringToReplace, replacement);
+}
+
+function addPathComponentToURL(url: string, pathComponent: string): string {
+  return url + "/" + pathComponent;
+}
+
+export function WorkflowActions({
+  workflow,
+  workflows,
+  repository,
+  branches,
+  defaultBranch,
+  favorites,
+  onUpdateFavorites,
+  mutateList,
+}: WorkflowActionsProps) {
+  const { octokit } = getGitHubClient();
+  const [owner, repo] = repository.split("/");
+
+  function favoriteActionTitle(workflow: Workflow, favorites: Workflow[]) {
+    return isFavoriteWorkflow(workflow, favorites) ? "Remove From Favorites" : "Add To Favorites";
+  }
+
+  function favoriteActionIcon(workflow: Workflow, favorites: Workflow[]) {
+    return isFavoriteWorkflow(workflow, favorites) ? Icon.StarDisabled : Icon.Star;
+  }
+
+  return (
+    <ActionPanel>
+      <Action.OpenInBrowser title="Open In Browser" url={createWorkflowURL(workflow)} />
+      <ActionPanel.Section>
+        <Action.Push
+          title="Run With Options"
+          icon={Icon.List}
+          target={<WorkflowForm repository={repository} workflow={workflow} branches={branches} />}
+          shortcut={{ modifiers: ["cmd"], key: "enter" }}
+        />
+        <Action
+          title="Run With Defaults"
+          icon={Icon.Clock}
+          shortcut={{ modifiers: ["cmd", "shift"], key: "enter" }}
+          onAction={() => {
+            runWorkflow(workflow, repository, defaultBranch, {}, octokit);
+          }}
+        />
+      </ActionPanel.Section>
+      <ActionPanel.Section>
+        <Action
+          title={favoriteActionTitle(workflow, favorites)}
+          icon={favoriteActionIcon(workflow, favorites)}
+          shortcut={{ modifiers: ["cmd", "ctrl", "shift"], key: "enter" }}
```

Comment:
The shortcut appears to be a bit complicated. What do you think of this one:

```suggestion
          shortcut={{ modifiers: ["cmd", "shift"], key: "f" }}
```

Reply:
Much better!